EnglishFilter/sections/antiadblock.txt

```diff
@@ -140,6 +140,7 @@ infotaxco.com,durtypass.com,dzapk.com,wikihandbk.com,freecourselab.com,onehack.u
 ! BAB 4.2b (`$$script[wildcard="*['length','charCodeAt','fromCharCode'];*(![]+[])*"][min-length="5000"][max-length="27000"]`)
 faucet.dog,dogestar.website,faucet.plus,crypto-faucet.xyz,alobuu.com,omgexploits.com,aloass.com,cryptslice.com,luutruhay.net,taisv.com,aloguy.com,alohdd.com,alogum.com,alobyt.com,aloboi.com,uebnews.online,aloegg.com,alofps.com,pennbookcenter.com,link1s.*,ltc24.com,jagoanssh.com,gainbtc.click,myreadingmanga.info,paidtomoney.com,jpopsingles.eu,pasend.*,freeomovie.to$$script[wildcard="*['length','charCodeAt','fromCharCode'];*(![]+[])*"][min-length="5000"][max-length="27000"]
 ! Old BAB (`$$script[wildcard="*eval*babasbmsgx*"][min-length="5000"][max-length="20000"]`)
+goldenmanga.top$$script[wildcard="*eval*babasbmsgx*"][min-length="5000"][max-length="20000"]
```

Comment:
You can just add the domain next to `racevpn.com,`.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/93669/commits/cf6a61ed4df5c2239e9e830af2c472db763e050c